### PR TITLE
Add license and autoupdate info of "protovis"

### PR DESCRIPTION
**Pull request by AilurusUmbra**
Add license of protovis ,Ref: http://protovis.org/
Add autoupdate field of protovis

Pull request for issue:  #12289 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/mbostock/protovis/tree/v3.3.0
 * Official website (optional, not the repository): http://protovis.org/
 * NPM package url (optional): none
 * License and its reference: BSD, Ref: http://protovis.org/
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 36
   - Count of stars: 603
   - Count of forks: 115
 * NPM download stats (optional): none
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@AilurusUmbra let give each commit their own meaningful commit message!

**Comment by sufuf3** [comment]
@AilurusUmbra you can refer to https://github.com/cdnjs/cdnjs/commits?author=AilurusUmbra this commit message.
Also, you can use `git rebase -i HEAD~2` to squash the commits to one commit.
And modify the commit message.

If you have any question, please let me know.

Thank you.

File: ajax/libs/protovis/package.json [hunk 1]
```diff
@@ -12,5 +12,18 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/mbostock/protovis"
+  },
+  "license": "BSD",
```
Comment (extend1994) [inline ajax/libs/protovis/package.json]: You can find the license link in http://protovis.org/ is BSD-2-Clause.
cdnjs needs `license` to be SPDX identifier. So let's modify this to `BSD-2-Clause`. 
You can refer https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json or https://spdx.org/licenses/.

**Comment by sashberd** [comment]
@extend1994 Help with squash and rebase to latest master
FYR

File: ajax/libs/protovis/package.json [hunk 2]
```diff
@@ -11,6 +11,19 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/mbostock/protovis"
+    "url": "https://mbostock.github.io/protovis/"
```
Comment (extend1994) [inline ajax/libs/protovis/package.json]: Please don't touch repo URL. The homepage is in line 6. Let's modify it there.

File: ajax/libs/protovis/package.json [hunk 3]
```diff
@@ -11,6 +11,19 @@
...
+  },
+  "license": "BSD-2-Clause",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/mbostock/protovis.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
```
Comment (extend1994) [inline ajax/libs/protovis/package.json]: Wrong auto-update config. Please revise it, thanks.

**Pull request closed without merging**